railties/lib/rails/tasks/statistics.rake

```diff
@@ -10,20 +21,13 @@ STATS_DIRECTORIES = [
   %w(Channels           app/channels),
   %w(JavaScripts        app/assets/javascripts),
   %w(Libraries          lib/),
-  %w(APIs               app/apis),
```

Comment:
It's still there, I think the diff is just pointing out that I removed the comma :)